chapter08/spg/exercise1.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my $file;
+my $string;
+
+print "Enter 1) for scalar filehandle, or 2) for file filehandle\n";

```

Comment:
I think you must have missed part of the question here - there was to be a 3rd option to write to both handles.  Either that or I'm a chump :smile: